test/zdtm/static/maps00.c

```diff
@@ -158,7 +158,8 @@ static int check_map(struct map *map)
 
 	if (!sigsetjmp(segv_ret, 1)) {
 		if (map->prot & PROT_WRITE) {
-			memcpy(map->ptr, test_func, getpagesize());
+			memcpy(map->ptr, test_func, ONE_MAP_SIZE);
+			__builtin___clear_cache(map->ptr, map->ptr + ONE_MAP_SIZE);
```

Comment:
pls add a comment why we need to cleare cache here

Reply:
done